microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/util/MicroProfileFaultToleranceServerConfiguration.java

```diff
@@ -0,0 +1,132 @@
+     *
+     * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
+     *         {@link OnlineManagementClient} API
+     */
+    public static void enableFaultTolerance() throws ManagementClientRelatedException {
+        try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            enableFaultTolerance(client);
+        } catch (Exception ex) {
+            throw new ManagementClientRelatedException(
+                    "Enabling Fault Tolerance failed. This might crash other tests.", ex);
+        }
+    }
+
+    /**
+     * Enable fault tolerance extension and subsystem.
+     *
+     * @param client {@link OnlineManagementClient} instance used to execute the command
+     * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
+     *         {@link OnlineManagementClient} API
+     */
+    public static void enableFaultTolerance(OnlineManagementClient client) throws ManagementClientRelatedException {
+        if (!faultToleranceExtensionExists(client)) {
+            executeCliCommand(client, MICROPROFILE_FAULT_TOLERANCE_EXTENSION_ADDRESS + ":add()");
+        }
+        if (!faultToleranceSubsystemExists(client)) {
+            executeCliCommand(client, MICROPROFILE_FAULT_TOLERANCE_SUBSYSTEM_ADDRESS + ":add()");
+        }
+        try {
+            new Administration(client).reloadIfRequired();
+        } catch (Exception ex) {
+            throw new ManagementClientRelatedException(
+                    "Enabling Fault Tolerance subsystem failed. This might crash other tests.", ex);
+        }
+    }
+
+    /**
+     * Disable Fault Tolerance subsystem and extension
+     *
+     * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
+     *         {@link OnlineManagementClient} API
+     */
+    public static void disableFaultTolerance() throws ManagementClientRelatedException {
+        try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            disableFaultTolerance(client);
+        } catch (Exception ex) {
+            throw new ManagementClientRelatedException(
+                    "Disabling Fault Tolerance subsystem failed. This might crash other tests.", ex);
+        }
+    }
+
+    /**
+     * Disable Fault Tolerance subsystem and extension
+     *
+     * @param client {@link OnlineManagementClient} instance used to execute the command
+     * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
+     *         {@link OnlineManagementClient} API
+     */
+    public static void disableFaultTolerance(OnlineManagementClient client) throws ManagementClientRelatedException {
+        if (faultToleranceSubsystemExists(client)) {
+            executeCliCommand(client, MICROPROFILE_FAULT_TOLERANCE_SUBSYSTEM_ADDRESS + ":remove()");
```

Comment:
Why are you using `executeCliCommand` here with `":remove()"` and you are calling `exists()` operation later (in `faultToleranceSubsystemExists`) through Creaper?

Reply:
Resolved on IRC discussion.

Reply:
fixed